db/migrate/20180601043148_create_tasks.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTasks < ActiveRecord::Migration
+  def change
+    create_table :tasks, id: :uuid, default: 'uuid_generate_v4()' do |t|
+      t.integer :state
+      t.string :name
+      t.text :description
+      t.uuid :project_id, index: true, foreign_key: true, default: 'uuid_generate_v4()'
```

Comment:
you don't need the default there as it's a foreign key so should be provided
I'm pretty sure this option is just ignored here.

Reply:
Thanks for the feedback!